### DISCUSSION TITLE
Can't delete Plays from Playbook

### DISCUSSION THREAD

**Discussion by hacker0ni** [Security-Onion-Solutions/securityonion#5997] [Unsupported Versions]
Hi! So while testing how to write custom Plays for the Playbook, I've created some Plays that I do not need anymore. When I click near the three dots on the play I want to delete, the `copy` and `delete` options are greyed out and I can not get rid of these plays (even though they're inactive). Is there a file I can edit manually to remove these plays? Or is there way I can achieve this in the UI? Thanks in advance!

**Comment by Sahale** [comment]
Hi!
You can delete them with admin account, password you can get with `# salt-call pillar.get secrets`
